src/binder_plugin.c

```diff
@@ -1615,6 +1617,47 @@ binder_plugin_create_slot(
         slot->data_opt.allow_data = ival;
     }
 
+    /* technologies */
+    strv = ofono_conf_get_strings(file, group, BINDER_CONF_SLOT_TECHNOLOGIES, ',');
+    if (strv) {
+        char **p;
+
+        config->techs = 0;
+        for (p = strv; *p; p++) {
+            const char *s = *p;
+            enum ofono_radio_access_mode m;
+
+            if (!s[0]) {
+                continue;
+            }
+
+            if (!strcmp(s, "all")) {
+                config->techs = OFONO_RADIO_ACCESS_MODE_ALL;
+                break;
+            }
+
+            if (!ofono_radio_access_mode_from_string(s, &m)) {
+                ofono_warn("Unknown technology %s in [%s] "
+                    "section of %s", s, group,
+                    BINDER_CONF_FILE);
+                continue;
+            }
+
+            if (m == OFONO_RADIO_ACCESS_MODE_ANY) {
+                config->techs = OFONO_RADIO_ACCESS_MODE_ALL;
+                break;
+            }
+
+            config->techs |= m;
+        }
+        g_strfreev(strv);
+    }
+
+    /* limit technologies based on radioInterface */
+    if (slot->version < RADIO_INTERFACE_1_4) {
+        config->techs &= ~OFONO_RADIO_ACCESS_MODE_NR;
```

Comment:
How about using `~` (rather than `-`) for consistency, even though it's practically the same thing for a signed int

Reply:
This is already using ~. What do you mean by your comment?

Reply:
oh, it could be that font smoothing and/or a too small font fooled me. never mind then